src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -3,7 +3,11 @@ import { Route, Switch, useHistory, useRouteMatch } from "react-router-dom";
 
 import { AnyVenue, VenueTemplate } from "types/venues";
 
-import { FriendShipPage } from "pages/FriendShipPage";
+import { AnnouncementMessage } from "components/molecules/AnnouncementMessage";
+
+import { ChatSidebar } from "components/organisms/ChatSidebar";
+import { UserProfileModal } from "components/organisms/UserProfileModal";
+import { WithNavigationBar } from "components/organisms/WithNavigationBar";
```

Comment:
The way the order of these imports was changed is backwards. From top to bottom it should be:

- `pages/*`
- `components/templates/*`
- `components/organsims/*`
- `components/molecules/*`
- `components/atoms/*`

As was detailed in my original review comment about this in https://github.com/sparkletown/sparkle/pull/1218#discussion_r594002774